src/pages/inspect/block/ui/block-summary.tsx

```diff
@@ -0,0 +1,103 @@
+import { InfoCard } from '@/pages/explore/ui/info-card';
+import { Text } from '@penumbra-zone/ui/Text';
+import { TableCell } from '@penumbra-zone/ui/TableCell';
+import { BlockSummaryApiResponse } from '@/shared/api/server/block/types';
+import { ValueViewComponent } from '@penumbra-zone/ui/ValueView';
+import { pnum } from '@penumbra-zone/types/pnum';
+import { Metadata } from '@penumbra-zone/protobuf/penumbra/core/asset/v1/asset_pb';
+
+export function BlockSummary({ blockSummary }: { blockSummary: BlockSummaryApiResponse }) {
+  if ('error' in blockSummary) {
+    return <div>Error: {blockSummary.error}</div>;
+  }
+
+  return (
+    <div>
+      <div className='grid grid-cols-1 tablet:grid-cols-2 desktop:grid-cols-3 gap-2 mb-8'>
+        <InfoCard title='Total Transactions'>
+          <Text large color='text.primary'>
+            {blockSummary.numTxs}
+          </Text>
+        </InfoCard>
+        <InfoCard title='Total Swaps'>
+          <Text large color='text.primary'>
+            {blockSummary.numSwaps}
+          </Text>
+        </InfoCard>
+        <InfoCard title='Total Swap Claims'>
+          <Text large color='text.primary'>
+            {blockSummary.numSwapClaims}
+          </Text>
+        </InfoCard>
+        <InfoCard title='Total Open LPs'>
+          <Text large color='text.primary'>
+            {blockSummary.numOpenLps}
+          </Text>
+        </InfoCard>
+        <InfoCard title='Total Closed LPs'>
+          <Text large color='text.primary'>
+            {blockSummary.numClosedLps}
+          </Text>
+        </InfoCard>
+        <InfoCard title='Total Withdrawn LPs'>
+          <Text large color='text.primary'>
+            {blockSummary.numWithdrawnLps}
+          </Text>
+        </InfoCard>
+      </div>
+      <div>
+        <div className='mb-4'>
+          <Text large color='text.primary'>
+            Swaps
+          </Text>
+        </div>
+        <div className='grid grid-cols-4'>
+          <div className='grid grid-cols-subgrid col-span-4'>
+            <TableCell heading>From</TableCell>
```

Comment:
question: how ok is the experience with this component after using it?

Reply:
I didn't try all the props, but it was a straight fwd drop-n-replace 👍